ingestion/functions/parsing/cuba/cuba.py

```diff
@@ -0,0 +1,181 @@
+            if 'diagnosticados' in json_data['casos']['dias'][day]:
+                fecha = json_data['casos']['dias'][day]['fecha']
+                for entry in json_data['casos']['dias'][day]['diagnosticados']:
+                    notes = []
+                    case = {
+                        "caseReference": {
+                            "sourceId": source_id,
+                            "sourceEntryId": entry["id"],
+                            "sourceUrl": source_url},
+                        "location": convert_location(entry),
+                        "events": [
+                            {
+                                "name": "confirmed",
+                                "dateRange": {
+                                    "start": convert_date(fecha),
+                                    "end": convert_date(fecha)}}],
+                        "demographics": {
+                            "ageRange": {
+                                "start": float(
+                                    entry["edad"]),
+                                "end": float(
+                                    entry["edad"])},
+                            "gender": convert_gender(
+                                entry["sexo"]),
+                            "nationalities": [
+                                convert_nationality(
+                                    entry['pais'])]}}
+
+                    if entry.get("consulta_medico", ""):
+                        case["events"].append({
+                            "name": "firstClinicalConsultation",
+                            "dateRange": {
+                                "start": convert_date(entry["consulta_medico"]),
+                                "end": convert_date(entry["consulta_medico"])
+                            }}
+                        )
+
+                    if entry.get('posible_procedencia_contagio', ""):
+                        if 'crucero' in entry['posible_procedencia_contagio']:
+                            case["transmission"] = {"places": "Cruise Ship"}
+
+                        elif len(entry['posible_procedencia_contagio'][0]) == 2:
+                            for two_letter_country_code in entry['posible_procedencia_contagio']:
+                                if pycountry.countries.get(
+                                        alpha_2=two_letter_country_code):
+                                    country = pycountry.countries.get(
+                                        alpha_2=two_letter_country_code).name
+                                    if country:
+                                        case["travelHistory"] = {
+                                            "traveledPrior30Days": True,
+                                            "travel": [
+                                                {
+                                                    "location": {
+                                                        "query": "Cuba"
```

Comment:
we don't set the last destination of travels in here, it is the case's location itself. travel.location should include the countries they travelled to before arriving in Cuba. If you just want to put that info in the notes as you did here then you can leave `travel.location` unset.

Reply:
PR looks good but please remove this and then ping us for merging into main